plugins/modules/system/alternatives.py

```diff
@@ -180,8 +213,13 @@ def main():
                 if not link:
                     module.fail_json(msg="Needed to install the alternative, but unable to do so as we are missing the link")
 
+                cmd = [UPDATE_ALTERNATIVES, '--install', link, name, path, str(priority)]
+                if params['slaves']:
+                    slaves = map(lambda slave: ['--slave', slave['link'], slave['name'], slave['path']], params['slaves'])
```

Comment:
How about using a list comprehension instead:
```suggestion
                    slaves = [['--slave', slave['link'], slave['name'], slave['path']] for slave in params['slaves']]
```